mappings/net/minecraft/datafixers/fixes/ItemOminousBannerTranslataionKeyRenameFix.mapping

```diff
@@ -0,0 +1 @@
+CLASS aae net/minecraft/datafixers/fixes/ItemOminousBannerTranslataionKeyRenameFix
```

Comment:
It's still called *Transalation* instead of *Translation*.

Reply:
no item prefix plz! just 

OminousBannerNameFix

Reply:
All other names in this package are prefixef